CITATION.cff

```diff
@@ -0,0 +1,157 @@
+# This file is a citation metadata file in the Citation File Format
+# (http://citation-file-format.github.io/).
+#
+# It provides citation metadata for this version of BOUT++
+
+cff-version: 1.0.3-1
+message: If you use BOUT++ in your work, please cite it using the following metadata,
+  or the two papers in the reference section
+title: BOUT++
+authors:
+  - family-names: Dudson
+    given-names: Benjamin Daniel
+    orcid: https://orcid.org/0000-0002-0094-4867
+    affiliation: University of York
+  - family-names: Hill
```

Comment:
Is it possible to have whitespace or something to separate the entries, it can be difficult to see where one person ends and another starts?

Reply:
Yes, good thinking